### ISSUE TITLE
latest Docker container missing various libraries

### ISSUE THREAD

**Issue by r0ckarong** [standardese/standardese#192]
I've tried to run 0.5.0 (that's what I assume is in the latest Docker image) and I get the error:

```
standardese@0bb1fbccb75f:/usr$ standardese --help
standardese: error while loading shared libraries: libstandardese.so: cannot open shared object file: No such file or directory
```

Checking the dependencies it looks like a few things like cppast, boost and the main library are not present in the container.

```
standardese@0bb1fbccb75f:/lib$ ldd /usr/local/bin/standardese
        linux-vdso.so.1 (0x00007ffe48b50000)
        libstandardese.so => not found
        libboost_program_options.so.1.71.0 => not found
        libboost_filesystem.so.1.71.0 => not found
        libboost_system.so.1.71.0 => not found
        libcppast.so => not found
        lib_cppast_tiny_process.so => not found
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007fdea4c66000)
        libclang-10.so.1 => /lib/x86_64-linux-gnu/libclang-10.so.1 (0x00007fdea2bbb000)
        libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007fdea29da000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007fdea288b000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007fdea2870000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007fdea267c000)
        /lib64/ld-linux-x86-64.so.2 (0x00007fdea4c91000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007fdea2676000)
        libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007fdea265a000)
        librt.so.1 => /lib/x86_64-linux-gnu/librt.so.1 (0x00007fdea264f000)
        libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007fdea261f000)
        libLLVM-10.so.1 => /lib/x86_64-linux-gnu/libLLVM-10.so.1 (0x00007fde9df8b000)
        libffi.so.7 => /lib/x86_64-linux-gnu/libffi.so.7 (0x00007fde9df7d000)
        libedit.so.2 => /lib/x86_64-linux-gnu/libedit.so.2 (0x00007fde9df45000)
        libbsd.so.0 => /lib/x86_64-linux-gnu/libbsd.so.0 (0x00007fde9df2b000)
```

**Comment by saraedum** [comment]
Thanks for reporting. This should be fixed in 0.5.1. The docker image for 0.5.1 should be available shortly. It would be great if you could check that this works now :)

**Comment by saraedum** [comment]
Btw., the automatic build of releases on docker hub was broken. In the future the images should be available again as standardese/standardese:0.5.1.
